File: VB/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports DevExpress.XtraPivotGrid
Imports DevExpress.XtraReports.UI
' ...

Namespace XRPivotGridEvents
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub xrPivotGrid1_PrintCell(ByVal sender As Object, ByVal e As DevExpress.XtraReports.UI.PivotGrid.CustomExportCellEventArgs) Handles xrPivotGrid1.PrintCell
			If e.ColumnIndex = 1 AndAlso Convert.ToDouble(e.Value) > 50 Then
				e.Brick.BackColor = Color.AliceBlue
			End If
		End Sub

		Private Sub xrPivotGrid1_PrintFieldValue(ByVal sender As Object, ByVal e As DevExpress.XtraReports.UI.PivotGrid.CustomExportFieldValueEventArgs) Handles xrPivotGrid1.PrintFieldValue
			If e.Text <> "Condiments" Then
				e.Brick.BackColor = Color.Beige
			End If
		End Sub

		Private Sub xrPivotGrid1_PrintHeader(ByVal sender As Object, ByVal e As DevExpress.XtraReports.UI.PivotGrid.CustomExportHeaderEventArgs) Handles xrPivotGrid1.PrintHeader
			If e.Caption = "Quantity" Then
				e.Brick.BackColor = Color.AliceBlue
			End If
		End Sub

	End Class
End Namespace

File: CS/XtraReport1.cs
using System;
using System.Drawing;
using DevExpress.XtraPivotGrid;
using DevExpress.XtraReports.UI;
// ...

namespace XRPivotGridEvents {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void xrPivotGrid1_PrintCell(object sender, DevExpress.XtraReports.UI.PivotGrid.CustomExportCellEventArgs e) {
            if (e.ColumnIndex == 1 && Convert.ToDouble(e.Value) > 50) {
                e.Brick.BackColor = Color.AliceBlue;
            }
        }

        private void xrPivotGrid1_PrintFieldValue(object sender, DevExpress.XtraReports.UI.PivotGrid.CustomExportFieldValueEventArgs e) {
            if (e.Text != "Condiments") {
                e.Brick.BackColor = Color.Beige;
            }
        }

        private void xrPivotGrid1_PrintHeader(object sender, DevExpress.XtraReports.UI.PivotGrid.CustomExportHeaderEventArgs e) {
            if (e.Caption == "Quantity") {
                e.Brick.BackColor = Color.AliceBlue;
            }
        }

    }
}
